04-mpu6050/scripts/temperature.sh

```diff
@@ -19,3 +19,15 @@ echo "Value of WHO_AM_I register is : $who_am_i"
 echo "Turning on MPU6050"
 $(i2cset -y 0 0x68 107 0)
 
+while [ true ]
+do
+        tempH=$(i2cget -y 0 0x68 65)
+        tempL=$(i2cget -y 0 0x68 66)
+        temp_abs=$(( -(((tempH << 8)+ tempL) ^ ((1 << 0x10) - 1)) ))
```

Comment:
Please check the correctness of this conversation. What will be in result for temperature above 36.53 C?